routes/configOverview.js

```diff
@@ -30,6 +30,28 @@ var route = function (req, res) {
 
         },
 
+        function (cb) {
```

Comment:
Make sure you test that this works for config as well. Config data is a bit different from models.